### ISSUE TITLE
`Fmt.add_rainbow_filters` results in an undefined method error.

### ISSUE THREAD

**Issue by KonnorRogers** [fractaledmind/prompts#8]
Method call looks like this:

```rb
        manifest_path = Prompts::TextPrompt.ask(
          label: "What is the file path of your custom elements manifest?",
          hint: "IE: node_modules/@shoelace-style/shoelace/dist/custom-elements.json",
          required: true
        )

```

Error:

```
➜  phlex_custom_element_generator git:(main) ✗ ./exe/phlex_custom_element_generator print_phlex_registrations
/Users/konnor/.asdf/installs/ruby/3.3.0/lib/ruby/gems/3.3.0/gems/prompts-0.2.1/lib/prompts.rb:8:in `<top (required)>': undefined method `add_rainbow_filters' for module Fmt (NoMethodError)

Fmt.add_rainbow_filters

        from <internal:/Users/konnor/.asdf/installs/ruby/3.3.0/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:132:in `require'
        from <internal:/Users/konnor/.asdf/installs/ruby/3.3.0/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:132:in `rescue in require'
        from <internal:/Users/konnor/.asdf/installs/ruby/3.3.0/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:126:in `require'
        from /Users/konnor/projects/phlex_custom_element_generator/lib/phlex_custom_element_generator/cli.rb:2:in `<top (required)>'
        from /Users/konnor/projects/phlex_custom_element_generator/lib/phlex_custom_element_generator.rb:4:in `require_relative'
        from /Users/konnor/projects/phlex_custom_element_generator/lib/phlex_custom_element_generator.rb:4:in `<top (required)>'
        from ./exe/phlex_custom_element_generator:3:in `require_relative'
        from ./exe/phlex_custom_element_generator:3:in `<main>'
<internal:/Users/konnor/.asdf/installs/ruby/3.3.0/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:127:in `require': cannot load such file -- prompts (LoadError)
        from <internal:/Users/konnor/.asdf/installs/ruby/3.3.0/lib/ruby/3.3.0/rubygems/core_ext/kernel_require.rb>:127:in `require'
        from /Users/konnor/projects/phlex_custom_element_generator/lib/phlex_custom_element_generator/cli.rb:2:in `<top (required)>'
        from /Users/konnor/projects/phlex_custom_element_generator/lib/phlex_custom_element_generator.rb:4:in `require_relative'
        from /Users/konnor/projects/phlex_custom_element_generator/lib/phlex_custom_element_generator.rb:4:in `<top (required)>'
        from ./exe/phlex_custom_element_generator:3:in `require_relative'
        from ./exe/phlex_custom_element_generator:3:in `<main>'
        
```

**Comment by KonnorRogers** [comment]
Solved. Just need to do:

```
require "rainbow" # this must come first
require "fmt"
```

and then swap to pipeline operators `|>red`

**Comment by BarnabeD** [comment]
Same issue for me after un bundle update.
Thanks for the PR @KonnorRogers. 
I hope it would be merged soon 😃

**Comment by fractaledmind** [comment]
Released with version 0.3.0
